docs/configuration/region-by-country.mdx

```diff
@@ -33,7 +33,8 @@ If you'd like to contribute information for your country, click the "Edit this p
 | -------------- | ------------------ | -------------------------------------------------------------------------------------------------------------------------------- |
 | Canada         | US                 | [LoRaWAN Regional Parameters](https://www.thethingsnetwork.org/docs/lorawan/regional-parameters/)                                |
 | China          | CN                 | [LoRaWAN Regional Parameters](https://lora-alliance.org/wp-content/uploads/2020/11/lorawan_regional_parameters_v1.0.3reva_0.pdf) |
-| Croatia        | EU_868<br />EU_433 |                                                                                                                                  |
+| Chile          | ANZ                | [LoRaWAN Regional Parameters](https://lora-alliance.org/wp-content/uploads/2020/11/lorawan_regional_parameters_v1.0.3reva_0.pdf) |
```

Comment:
Just realized these are for Lorawan...not sure what we want to do here. @rcarteraz @pdxlocations